### PR TITLE
CEDS-3061 Add navigation banner

### PR DESCRIPTION
**Pull request by MaciejRewera**
This component is present on 3 pages:
- inbox_wrapper
- conversation_wrapper
- upload_your_files_confirmation

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -6,6 +6,9 @@ common.continue=Continue
 common.feedback=This is a new service - your {0} will help us to improve it.
 common.feedback.link=feedback
 
+common.navigation.messages = Messages
```
Comment (udinobi) [inline conf/messages:9]: `Messages` or `Conversations` ? It seems our dictionary is not always consistent. Sometime we use _message_ and other _conversation_.

File: test/views/UploadYourFilesConfirmationSpec.scala [hunk 2]
```diff
@@ -16,22 +16,41 @@
...
-  val view: Document = page(List(sampleFileUpload), Some(mrn), email)(fakeRequest, messages)
+  override def beforeEach(): Unit = {
+    super.beforeEach()
+
+    reset(secureMessagingConfig)
+    when(secureMessagingConfig.isSecureMessagingEnabled).thenReturn(true)
+  }
+
+  override def afterEach(): Unit = {
+    reset(secureMessagingConfig)
```
Comment (udinobi) [inline test/views/UploadYourFilesConfirmationSpec.scala:51]: It does not seem really necessary.

File: test/views/components/gds/MainTemplateSpec.scala [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  private def createView(withNavigationBanner: Boolean): Document =
+    mainTemplate(title = Title("common.service.name"), withNavigationBanner = withNavigationBanner)(testContent)(fakeRequest, messages)
+
+  override def beforeEach(): Unit = {
+    super.beforeEach()
+    reset(secureMessagingConfig)
+  }
+
+  override def afterEach(): Unit = {
+    reset(secureMessagingConfig)
```
Comment (udinobi) [inline test/views/components/gds/MainTemplateSpec.scala:47]: It does seem unnecessary.

**Review by glidester — APPROVED** [review]
LGTM

File: conf/messages (hunk 1 above)
Comment (MaciejRewera) [reply conf/messages:9]: This is taken from the prototype. I am not sure if Conversations and Messages are the same concepts.

**Pull request merged**